tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -138,12 +141,42 @@ public static void initTrace(Optional<Boolean> isObservable, String traceId) {
                 .map(observable -> observable ? Observability.SAMPLE : Observability.DO_NOT_SAMPLE)
                 .orElse(Observability.UNDECIDED);
 
-        setTrace(createTrace(observability, traceId));
+        setTrace(createTrace(observability, traceId, Optional.empty()));
     }
 
-    /** Initializes the current thread's trace, erasing any previously accrued open spans. */
+    /**
+     * Initializes the current thread's trace, erasing any previously accrued open spans.
+     *
+     * @deprecated Use {@link #initTraceWithSpan(Observability, String, String, SpanType)}
+     */
+    @Deprecated
     public static void initTrace(Observability observability, String traceId) {
```

Comment:
There are a few usages of this method left in our tests, can we get rid of them as well?

Reply:
Ah you're right, they're left in tests that suppress deprecation :-/